apps/web/src/pages/Applications/ApplicantDashboardPage/components/DashboardHeading.tsx

```diff
@@ -53,6 +63,31 @@ const DashboardHeading = ({ user }: DashboardHeadingProps) => {
   const intl = useIntl();
   const paths = useRoutes();
 
+  const notEmptyExperiences = user.experiences?.filter(notEmpty);
+
+  const awardExperiences =
+    notEmptyExperiences
+      ?.filter(isAwardExperience)
+      .map(
+        (award: AwardExperience) =>
+          ({
+            ...award,
+            startDate: award.awardedDate,
+            endDate: award.awardedDate,
+          } as AwardExperience & { startDate: string; endDate: string }),
+      )
+      .sort(compareByDate) || [];
+  const communityExperiences =
+    notEmptyExperiences?.filter(isCommunityExperience).sort(compareByDate) ||
+    [];
+  const educationExperiences =
+    notEmptyExperiences?.filter(isEducationExperience).sort(compareByDate) ||
+    [];
+  const personalExperiences =
+    notEmptyExperiences?.filter(isPersonalExperience).sort(compareByDate) || [];
+  const workExperiences =
+    notEmptyExperiences?.filter(isWorkExperience).sort(compareByDate) || [];
```

Comment:
If we are only ever using the length, I think we can safely remove the `.sort(compareByDate)`

Reply:
Given the 🪓 in 335f5db63b2632e8aefed100dfc0b5597628ae0d.